src/sweeper/__main__.py

```diff
@@ -37,12 +41,12 @@ def main():
         print(args)
     elif args['empties']:
         print(args)
-
+        # initialize the class object
```

Comment:
comments should use the `#:` format so you can differentiate between a comment and commented out code.

Reply:
makes sense.  thx for the clarification.